source/docs/networking/networking-introduction/roborio-network-troubleshooting.rst

```diff
@@ -1,61 +1,101 @@
+
+Ping fails
+----------
+
+.. tabs::
+
+   .. group-tab:: Windows 10
+
+      .. image:: images/roborio-troubleshooting/win10-dhcp.png
+         :alt: Windows 10 image of the adapter setting
+
+      If pinging the IP address directly fails, you may have an issue with the network configuration of the PC. The PC should be configured to **Automatic**. To check this, click :guilabel:`Start` -> :guilabel:`Settings` -> :guilabel:`Network & Internet`. Depending on your network, select :guilabel:`Wifi` or :guilabel:`Ethernet`. Then click on your connected network. Scroll down to **IP settings** and click :guilabel:`Edit` and ensure the :guilabel:`Automatic (DHCP)` option is selected.
+
+   .. group-tab:: Windows 7
+
+      .. image:: images/roborio-troubleshooting/control-panel-dhcp.png
+         :alt: Windows 7 image of the adapter setting
+
+      If pinging the IP address directly fails, you may have an issue with the network configuration of the PC. The PC should be configured to **Obtain an Address Automatically** (also known as DHCP). To check this, click :guilabel:`Start` -> :guilabel:`Control Panel` -> :guilabel:`Network Connections` -> :guilabel:`Change adapter settings`, then right click on the appropriate interface (usually Local Area Connection for Ethernet or Wireless Network Connection for wireless) and select :guilabel:`Properties`. Click :guilabel:`Internet Protocol Version 4`, then click :guilabel:`Properties`. Make sure both radio buttons are set to :guilabel:`Obtain automatically`.
+
 USB Connection Troubleshooting
-------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If you are attempting to troubleshoot the USB connection, try pinging the roboRIO's IP address. As long as there is only one roboRIO connected to the PC, it should be configured as 172.22.11.2. If this ping fails, make sure you have the roboRIO connected and powered, and that you have installed the NI FRC Game Tools. The game tools installs the roboRIO drivers needed for the USB connection.
 
 If this ping succeeds, but the .local ping fails, it is likely that either the roboRIO hostname is configured incorrectly, or you are connected to a DNS server which is attempting to resolve the .local address.
 
 - Verify that your roboRIO has been :ref:`imaged for your team number <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`. This sets the hostname used by mDNS.
-- Disconnect your computer from all other networks including Ethernet and WiFi. It is possible that one of these networks contains a DNS server that is attempting to resolve the .local address.
+- :ref:`Disable all other network adapters <docs/networking/networking-introduction/roborio-network-troubleshooting:Disabling Network Adapters>`
 
 Ethernet Connection
```

Comment:
This also shouldn't be a sub-topic.

Reply:
I don't think I understand you comment.  Are you saying you thing the Network Adapters stuff should have its own page?

Reply:
Sorry, this was supposed to point at Ethernet Connection. Ethernet Connection should stay a higher level article `-`